core/app/models/spree/address.rb

```diff
@@ -15,6 +15,7 @@ class Address < Spree::Base
     alias_attribute :last_name, :lastname
 
     DB_ONLY_ATTRS = %w(id updated_at created_at)
+    TAX_ZONE_ONLY_ATTRS = %w(state_id country_id zipcode)
```

Comment:
This is a naming thing: In my opinion, the `tax_zone` is a weird implementation-specific artifact. Can we name this constant `TAXATION_ATTRIBUTES` or something like that that does not imply `Spree::Zone` to be involved? Because it isn't (directly)